### ISSUE TITLE
Movie recording of remote (Fastcamera) MotioneyeOS Cam isn't working

### ISSUE THREAD

**Issue by helmar74** [ccrisan/motioneyeos#1254]
Hi

I've encountered the following issue:
I have a motioneyeOS RPi3 running (without a cam). It's a Server for several other MotioneyeOS and "normal" Network Cams.

I set up a RPI Zero W with MotionEyeyOS. With normal setup I only got low framerates. I decided to switch over in "Fast camera mode" which brought me better framerates.

Than I connected the Fastcam to the central server instance and configured motion detections there.

Now following issue occurs:
If a motion is detected a new movie is created on the server, but the movie "hangs". That means that only the first picture of the movie is shown. The time in the right lower corner hangs on the time the movie was created. e.g. a movie is created at 19:00:00 and the movie is 30 seconds long. The time 19:00:00 is shown as long the movie is playing. it isn't moved on.

Has anyone an idea what's going wrong there?

Here's the config of the remote cam in central server:
![image](https://user-images.githubusercontent.com/25499089/33266351-8cfb8682-d375-11e7-848f-62aa44f35b65.png)
![image](https://user-images.githubusercontent.com/25499089/33266369-a47e3dc2-d375-11e7-84e3-d89e1bed77ca.png)

**Comment by alexleekt** [comment]
Is this not expected? https://github.com/ccrisan/motioneyeos/wiki/Fast-Network-Camera

**Comment by helmar74** [comment]
I don't think so, as I don't want to record on the Fast Network Camera itself. It's connected to a separeate MotionEyeOS Server as a "Network Camera". On this server I turned on the options above. A movie is recorded, when an motion event occurs. But the movie "hangs" it only shows one picture and not a movie

**Comment by jasaw** [comment]
@helmar74 Try changing the "Movie Format" to H264 without the OMX and please report back.

**Comment by helmar74** [comment]
Hi, I tried it.
Same result. Hangig "first" picture in the video, the time in the lower right corner always shows the same timestamp, no running movie
